### PR TITLE
Added star project request

### PR DESCRIPTION
**Pull request by iambami**
Signed-off-by: iambami <titatinuks@gmail.com>

**Description**

This PR fixes # 

**Notes to reviewers**

#### Checklist

- [] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -40,4 +40,13 @@ sentimentBotToxicityThreshold: .9
 # *Required* Comment to reply with
 sentimentBotReplyComment: >
   Please be sure to review the code of conduct and be respectful of other users. // @layer5io/maintainers 
-  
+
+
+
+
+#------------------------------------------------------------------------------------
```
Comment (leecalcote) [inline .github/config.yml:47]: Does this work?

This config should be under the #17 stanza, right?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thanks for taking this on, @iambami

File: .github/config.yml (hunk 1 above)
Comment (PluckyPrecious) [reply .github/config.yml:47]: You're right @leecalcote , Hi @iambami can you kindly take some time and effect the change and we will be good to go? 😃  The config should l look like this one https://github.com/layer5io/layer5/pull/723/commits/535e1ee3e10d9a6c4bd896f67e703e45ff923749#diff-51ae5ef21b0bd169a283475e122567d3

**Comment by PluckyPrecious** [comment]
Fixes #35

**Review by leecalcote — APPROVED** [review]
Thank you, @iambami. Looks good.

**Pull request merged**